### PR TITLE
Updates to chapter 8 for new interpolation variable

### PR DESCRIPTION
**Pull request by AndersMS**
Dear All,

Here is the restructuring of Chapter 8 discussed at the meting earlier this week. It reflects that we have moved attributes from the data variable to the interpolation variable.

Please note that the CDL examples have changed significantly.

Please review and let me know your comments.

Anders

### CODE REVIEW

**Comment by erget** [comment]
BTW merge conflict on the SVG is to be expected, it'll take some git magic and I'm ready to help after the text review.

File: ch08.adoc [hunk 1]
```diff
@@ -432,112 +435,52 @@ Note that an implementation of the interpolation method is free to calculate the
...
-  // Interpolation coefficient and configuration variables
-  short expansion1(zone_track , tp_scan) ;
-  short alignment1(zone_track , tp_scan) ;
-  short expansion2(tp_track, zone_scan) ;
-  short alignment2(tp_track, zone_scan) ;
+    tp_interpolation:interpolation_name = "bi_quadratic_remote_sensing" ;
+    tp_interpolation:tie_point_dimensions = "track: tp_track zone_track
+                                             scan: tp_scan zone_scan
+                                             scan: tp_time_scan"  ;
+    tp_interpolation:tie_point_indices = "track: track_indices zone_track
```
Comment (oceandatalab) [inline ch08.adoc]:
```suggestion
    tp_interpolation:tie_point_indices = "track: track_indices
```

File: ch08.adoc [hunk 2]
```diff
@@ -432,112 +435,52 @@ Note that an implementation of the interpolation method is free to calculate the
...
+                                          scan: scan_indices zone_scan
```
Comment (oceandatalab) [inline ch08.adoc]:
```suggestion
                                          scan: scan_indices
```

File: ch08.adoc [hunk 3]
```diff
@@ -432,112 +435,52 @@ Note that an implementation of the interpolation method is free to calculate the
...
-
-  // Interpolation coefficient and configuration variables
-  short expansion1(zone_track , tp_scan) ;
-  short alignment1(zone_track , tp_scan) ;
-  short expansion2(tp_track, zone_scan) ;
-  short alignment2(tp_track, zone_scan) ;
+    tp_interpolation:interpolation_name = "bi_quadratic_remote_sensing" ;
+    tp_interpolation:tie_point_dimensions = "track: tp_track zone_track
+                                             scan: tp_scan zone_scan
+                                             scan: tp_time_scan"  ;
```
Comment (oceandatalab) [inline ch08.adoc]: (Comment A)

```suggestion
;
```
`t` is the only tie point variable spanning the `tp_time_scan` dimension and there is no data variable in the CDL stating that `t` must be interpolated using `tp_interpolation` (they all use `time_interpolation`), so I am not sure if defining this mapping here is relevant.

File: ch08.adoc [hunk 4]
```diff
@@ -432,112 +435,52 @@ Note that an implementation of the interpolation method is free to calculate the
...
+    tp_interpolation:tie_point_indices = "track: track_indices zone_track
+                                          scan: scan_indices zone_scan
+                                          scan: time_scan_indices" ;
```
Comment (oceandatalab) [inline ch08.adoc]:
```suggestion
;
```
see (Comment A)

File: ch08.adoc (hunk 3 above)
Comment (AndersMS) [reply ch08.adoc]: Valid point, thank you for the careful review :-)

File: ch08.adoc [hunk 5]
```diff
@@ -432,112 +435,52 @@ Note that an implementation of the interpolation method is free to calculate the
...
+    I01_reflectance:units = "1" ;
+
+  float I04_radiance(track, scan) ;
+    I04_radiance:tie_points = "lat: lon: tp_interpolation  t: time_interpolation" ;
+    I04_radiance:standard_name = "toa_outgoing_radiance_per_unit_wavelength" ;
+    I04_radiance:units = "W m-2 sr-1 m-1" ;
+  float I04_brightness_temperature(track, scan) ;
+    I04_brightness_temperature:tie_points = "lat: lon: tp_interpolation  t: time_interpolation" ;
+    I04_brightness_temperature:standard_name = "brightness_temperature" ;
+    I04_brightness_temperature:units = "K" ;
```
Comment (davidhassell) [inline ch08.adoc:467]: Do we need four data variables in this example? They all have the span the same dimensions and have the same `tie_points` attribute value ....

**Pull request merged**